components/SongBar.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import React from "react";
+
+import PlayPause from "./PlayPause";
+import { Song } from "@/types";
+
+interface Props {
+  song: Song;
+  i: number;
+  isPlaying?: boolean;
+  activeSong?: Song;
+  handlePauseClick: () => void;
+  handlePlayClick: (song: Song, i: number) => void;
+}
+
+const SongBar: React.FC<Props> = ({
+  song,
+  i,
+  isPlaying,
+  activeSong,
+  handlePauseClick,
+  handlePlayClick,
+}) => (
+  <div
+    className={`w-full flex flex-row items-center hover:bg-[#4c426e] ${
+      activeSong?.title === song?.title ? "bg-[#4c426e]" : "bg-transparent"

```

Comment:
use [classNames](https://www.npmjs.com/package/classnames) to combine fixed with optional/dynamic class names.